pytorch_lightning/loops/optimization/optimizer_loop.py

```diff
@@ -426,14 +427,12 @@ def _training_step(self, split_batch: Any, batch_idx: int, opt_idx: int) -> Clos
             )
 
             # manually capture logged metrics
-            lightning_module._current_fx_name = "training_step"
-            with self.trainer.profiler.profile("training_step"):
-                training_step_output = self.trainer.accelerator.training_step(step_kwargs)
-                self.trainer.training_type_plugin.post_training_step()
+            training_step_output = self.trainer._call_accelerator_hook("training_step", step_kwargs)
+            self.trainer.training_type_plugin.post_training_step()
 
             del step_kwargs
 
-            training_step_output = self.trainer.call_hook("training_step_end", training_step_output)
+            training_step_output = self.trainer._call_lightning_module_hook("training_step_end", training_step_output)
```

Comment:
This is missing the TTP call and output selection. Should match what you have already in manual_loop